[2025-07-07T20:40:13.243+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: axone_fb_etl_pipeline.extract_facebook_data manual__2025-07-07T20:26:54.288448+00:00 [queued]>
[2025-07-07T20:40:13.289+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: axone_fb_etl_pipeline.extract_facebook_data manual__2025-07-07T20:26:54.288448+00:00 [queued]>
[2025-07-07T20:40:13.292+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 4
[2025-07-07T20:40:13.387+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): extract_facebook_data> on 2025-07-07 20:26:54.288448+00:00
[2025-07-07T20:40:13.493+0000] {standard_task_runner.py:57} INFO - Started process 244 to run task
[2025-07-07T20:40:13.523+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'axone_fb_etl_pipeline', 'extract_facebook_data', 'manual__2025-07-07T20:26:54.288448+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpqusdof_p']
[2025-07-07T20:40:13.529+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask extract_facebook_data
[2025-07-07T20:40:13.971+0000] {task_command.py:415} INFO - Running <TaskInstance: axone_fb_etl_pipeline.extract_facebook_data manual__2025-07-07T20:26:54.288448+00:00 [running]> on host c88dda942e9f
[2025-07-07T20:40:14.169+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='axone_fb_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_facebook_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T20:26:54.288448+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T20:26:54.288448+00:00'
[2025-07-07T20:40:14.174+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-07T20:40:14.177+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /host/project && python src/extract.py --query "deces jacques chirac" --num_scrolls 15']
[2025-07-07T20:40:14.196+0000] {subprocess.py:86} INFO - Output:
[2025-07-07T20:40:14.223+0000] {subprocess.py:93} INFO - /bin/bash: line 1: cd: /host/project: No such file or directory
[2025-07-07T20:40:14.224+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-07-07T20:40:14.246+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-07-07T20:40:14.254+0000] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=axone_fb_etl_pipeline, task_id=extract_facebook_data, execution_date=20250707T202654, start_date=20250707T204013, end_date=20250707T204014
[2025-07-07T20:40:14.270+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 10 for task extract_facebook_data (Bash command failed. The command returned a non-zero exit code 1.; 244)
[2025-07-07T20:40:14.304+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-07-07T20:40:14.380+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
